### PR TITLE
Update tutorials

### PR DESCRIPTION
**Pull request by cameronmartino**
Added three tutorials on the exact same time series IBD dataset but using:

- Python API
- QIIME2 API
- QIIME2 CLI

### CODE REVIEW

File: ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+Repeat measure experimental designs (e.g. time series) are valid and powerful method to control inter-individual variation. However, conventional dimensionality reduction methods can not account for the high-correlation of each subject to thierself at a later time point. This inherent correlation structure can cause subject grouping to confound or even outweigh important phenotype groupings. To address this we will use Compositional Tensor Factorization (CTF) which we provide in the software package [gemelli](https://github.com/cameronmartino/gemelli). CTF can account for repeated measures, compositionality, and sparsity in microbiome data.
```
Comment (gwarmstrong) [inline ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md:1]:
```suggestion
Repeat measure experimental designs (e.g. time series) are valid and powerful method to control inter-individual variation. However, conventional dimensionality reduction methods can not account for the high-correlation of each subject to theirself at a later time point. This inherent correlation structure can cause subject grouping to confound or even outweigh important phenotype groupings. To address this we will use Compositional Tensor Factorization (CTF) which we provide in the software package [gemelli](https://github.com/cameronmartino/gemelli). CTF can account for repeated measures, compositionality, and sparsity in microbiome data.
```

Comment (gwarmstrong) [reply ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md:1]: minor typo

Comment (cameronmartino) [reply ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md]: Resolved in #32

**Comment by cameronmartino** [comment]
@gwarmstrong Okay, this should be good to go now.

**Pull request merged**